addons/account/static/src/components/systray_item/systray_item.js

```diff
@@ -0,0 +1,29 @@
+/** @odoo-module **/
+
+import { registry } from "@web/core/registry";
+import { useService } from "@web/core/utils/hooks";
+import { Component } from "@odoo/owl";
+import { session } from '@web/session';
+
+export class AccountSystrayItem extends Component {
+    static template = "account.SystrayItem";
+    static props = {};
+
+    setup() {
+        this.currentCompany = useService("company").currentCompany;
+        this.isQuickEditModeEnabled = session.is_quick_edit_mode_enabled;
+    }
+}
+
+AccountSystrayItem.template = "account.SystrayItem";
+AccountSystrayItem.props = {};
+
```

Comment:
I guess you forgot to remove this.

Reply:
Sorry I indeed forgot it. It is fixed now